roles/generate-jenkins/templates/Jenkinsfile.j2

```
@@ -414,6 +415,32 @@ pipeline {
               else
                 echo "Jenkinsfile is up to date."
               fi
+              # Stage 2 - Delete old templates
+              OLD_TEMPLATES=".github/ISSUE_TEMPLATE.md"
+              for i in ${OLD_TEMPLATES}; do
+                if [[ -f "${i}" ]]; then
+                  TEMPLATES_TO_DELETE="${i} ${TEMPLATES_TO_DELETE}"
+                fi
+              done
+              if [[ -n "${TEMPLATES_TO_DELETE}" ]]; then
+                mkdir -p ${TEMPDIR}/repo
+                git clone https://github.com/${LS_USER}/${LS_REPO}.git ${TEMPDIR}/repo/${LS_REPO}
+                cd ${TEMPDIR}/repo/${LS_REPO}
+                git checkout -f {{ ls_branch }}
+                cd ${TEMPDIR}/docker-${CONTAINER_NAME}
+                for i in ${TEMPLATES_TO_DELETE}; do
+                  git rm "${i}"
```

```suggestion
                  git rm "${i}" || :
```

Does this need an "or true" of sorts? https://github.com/linuxserver/docker-jenkins-builder/pull/90/files#diff-e6ffa5dc854b843b3ee3c3c28f8eae2f436c2df2b1ca299cca1fa5982e390cf8L252 seems to have it.

---

Not needed because the file's existence is checked right above, before the var is set